wiki/Performance_points/de.md

```diff
@@ -0,0 +1,132 @@
+# Performance-Points
+
+**Performance Points** (oder **pp** abgekürzt, auch Performance-Punkte genannt) ist eine Metrik, deren Ziel es ist, den Spielerfortschritt in osu! kontextbezogener zu gestalten.
+
+Das Ziel ist es, den Schwerpunkt des Fertigkeitsfortschritts von der Spielzeit auf eine tatsächliche Darstellung der Fertigkeiten des Spielers zu verlagern. Dies wird durch die Berechnung einer einzigartigen Punktzahl erreicht, welche auf der Schwierigkeit der [Beatmap](/wiki/de/Beatmap) und der Leistung des Spielers basiert.
+
+## Geschichte
+
+Die erste Einführung so einer Punktzahl wurde im April 2012 der Öffentlichkeit bekanntgegeben und war nur unter dem mysteriösen Projekt *'???'* bekannt. Das rätselhafte System hat später im Monat seinen vollen Namen bekommen.
+
+Danach bekannt als "pp" (eine Abkürzung für "performance points"), versuchte dieses neue System, den alten Standard, der nur auf der [Gesamtpunktzahl](/wiki/Gameplay/Score) eines Spielers beruhte, zu etwas, dass die Fähigkeiten des Spielers widerspiegelt, zu verbessern. Das neue System wurde damals von den Spielern gelobt.
+
+Ein paar Monate nach der Enthüllung wurde es in der osu!-Version 20120722-24 offiziel eingebaut, um das alte [Ranglistenpunktesystem](/wiki/Beatmap/Category#ranked) vollständig zu ersetzen. Alle 30 Minuten wurden neue Scores berechnet. Später, im August desselben Jahres, wurde das System verbessert, sodass es die Performance-Points in Echtzeit aktualisierte.
+
+*Anmerkung: ppv1, die originale Implementierung für Performance Points, hatte auch ein Änderungsprotokoll, welches in seinem [Forumsbeitrag](https://osu.ppy.sh/community/forums/topics/92185) einsehbar ist.*
```

Comment:
```suggestion
*Tipp: ppv1, die originale Implementierung für Performance Points, hatte auch ein Änderungsprotokoll, welches in seinem [Forumsbeitrag](https://osu.ppy.sh/community/forums/topics/92185) einsehbar ist.*
```
Kinda nitpicky here but the so called [notices](https://osu.ppy.sh/wiki/de/Article_styling_criteria/Formatting#hinweis) are defined in the ASC, and thus this should require a rule change and an adjustment of all articles that are affected by this. Despite this, I'm not necessarily disagreeing to changing all `Tipp`s to `Anmerkung`en.

This was already explained in my [opening comment](https://github.com/ppy/osu-wiki/pull/7821#pullrequestreview-1071255102).

Reply:
Mind if I prepare a PR adjusting the ASC and occurences? Or 2 PRs I guess.

Reply:
Sure, go ahead if you want to. Not sure how many changes this would be exactly. My search tool showed around 30 articles when looking for `Tipp`.